theseus/src/api/profile.rs

```diff
@@ -20,6 +20,7 @@ pub async fn add(profile: Profile) -> crate::Result<()> {
     let state = State::get().await?;
     let mut profiles = state.profiles.write().await;
     profiles.insert(profile)?;
+    //State::sync().await?;
```

Comment:
Is this supposed to be commented out? (I assume the idea is to be adding State::sync() to the API funcs themselves and removing from the hook functions)

Reply:
Not sure why I commented it out- fixed